bin/sce_to_anndata.R

```diff
@@ -114,12 +122,19 @@ sample_metadata <- metadata(sce)$sample_metadata
 # make main sce czi compliant
 sce <- format_czi(sce)
 
+if (opt$compress_output) {
+  compression <- "gzip"
+} else {
+  compression <- "none"
+}
+
 # export sce as anndata object
 # this function will also remove any R-specific object types from the SCE metadata
 #   before converting to AnnData
 scpcaTools::sce_to_anndata(
   sce,
-  anndata_file = opt$output_rna_h5
+  anndata_file = opt$output_rna_h5,
+  compression = compression
```

Comment:
I think I would just inline this for directness
```suggestion
  compression = ifelse(opt$compress_output, "gzip", "none")
```